src/c++/perf_analyzer/genai-pa/genai_pa/llm_inputs/llm_inputs.py

```diff
@@ -54,6 +56,12 @@ class LlmInputs:
 
     DEFAULT_TRTLLM_MAX_TOKENS = 256
 
+    DEFAULT_RANDOM_SEED = 0
+    DEFAULT_PROMPT_TOKENS_MEAN = 550
+    DEFAULT_PROMPT_TOKENS_STDDEV = 250
+    DEFAULT_EXPECTED_OUTPUT_TOKENS = 150
+    DEFAULT_OUTPUT_PROMPTS = 10
```

Comment:
this is how many are generated? Seems like we would want 100? I don't think we're going to get much of a distribution at 10